### PR TITLE
Translation #6 - 1211 Words

### PR DESCRIPTION
**Pull request by cherryloua**
Hi everyone!

Please, take some time to review my PR. Thank you, - Cherryloua

### CODE REVIEW

File: book/08-customizing-git/sections/attributes.asc [hunk 1]
```diff
@@ -64,31 +64,32 @@ Ano ang ``word'' na pagsasala?
...
 
 [source,console]
 ----
 #!/bin/bash
 docx2txt.pl "$1" -
 ----
 
-Don't forget to `chmod a+x` that file.
-Finally, you can configure Git to use this script:
+Huwag kalimotan na i-`chmod a+x` ang file na iyon.
```
Comment (jjrang11) [inline book/08-customizing-git/sections/attributes.asc]: `kalimotan` to `kalimutan`

File: book/08-customizing-git/sections/attributes.asc [hunk 2]
```diff
@@ -98,36 +99,38 @@ index 0b013ca..ba25db5 100644
...
- What is "version control", and why should you care? Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later. For the examples in this book you will use software source code as the files being version controlled, though in reality you can do this with nearly any type of file on a computer.
+ Ang kabanatang ito ay tungkol sa pagsisimula sa Git. Magsisimula tayo sa simula sa pamamagitan ng pagpapaliwanag ng ilang mga background sa mga kasangkapan ng pagkontrol sa bersyon, pagkatapos ay lilipat sa kung paano makakuha ng Git na tumatakbo sa iyong sistema at sa panghuli kung paano makakuha ng setup na ito upang simulan ang pakikipagtulungan. Sa dulo ng kabanatang ito
+dapat mong maintindihan kung bakit ang Git ay nandito, bakit dapat mong gamitin ito at dapat mong i-setup ang lahat upang magawa ito.
+1.1. Tungkol sa Pagkontrol ng Bersyon
+Ano ang "pagkontrol sa bersyon", at bakit dapat kang makialam? Pagkontrol sa bersyon ay isang sistema na nagtatala sa mga pagbabago sa isang file o hanay ng mga file sa paglipas ng panahon upang maaari mong iurong ang mga tiyak na mga bersyon pagkalipas.
```
Comment (jjrang11) [inline book/08-customizing-git/sections/attributes.asc]: `Pagkontrol sa bersyon` to `Ang pagkontrol sa bersyon`

File: book/08-customizing-git/sections/attributes.asc [hunk 3]
```diff
@@ -189,31 +193,32 @@ $ cat test.txt
...
 
 [[filters_b]]
-.The ``clean'' filter is run when files are staged.
-image::images/clean.png[The ``clean'' filter is run when files are staged.]
+.Ang ``clean'' na pagsasala ay pinapatakbo kung ang mga file ay naka-stage.
+image::images/clean.png[Ang ``clean'' na pagsasala ay pinapatakbo kung ang mga file ay naka-stage.]
 
-The original commit message for this feature gives a simple example of running all your C source code through the `indent` program before committing.
-You can set it up by setting the filter attribute in your `.gitattributes` file to filter `*.c` files with the ``indent'' filter:
+Ang orihinal na mensahe ng commit para sa katangian na ito ay nagbibigay ng isang simpleng halimbawa ng pagpapatakbo ng lahat ng iyong source code na C sa pamamagitan ng `indent` na programa bago i-commit.
```
Comment (jjrang11) [inline book/08-customizing-git/sections/attributes.asc]: `source code na C` to `source code sa C`

**Review by jjrang11 — COMMENTED** [review]
hello there @cherryloua, great contribution! i just have a few suggestions.

**Comment by cherryloua** [comment]
Hi everyone,

I just updated the suggested changes. Thank you!

**Pull request merged**